objects/_objects.layout.scss

```diff
@@ -211,6 +211,10 @@ $inuit-use-markup-fix: false !default;
     display: flex;
   }
 
+  &.o-layout--center {
+    justify-content: center;
+  }
```

Comment:
My intention here was to preserve the original behavior for when using `.o-layout--center` by itself. The stretch modifier stretches in a vertical way, so a center alignment can be necessary at times.